types/src/unit_tests/trusted_state_test.rs

```diff
@@ -402,21 +402,29 @@ proptest! {
         let initial_li_with_sigs = lis_with_sigs.remove(0);
         let initial_li = initial_li_with_sigs.ledger_info();
         let trusted_state = TrustedState::try_from(initial_li).unwrap();
+        let good_li = latest_li.ledger_info();
 
         // Verifying a latest ledger info (inside the last epoch) with invalid
-        // signatures should fail.
-        let latest_li = LedgerInfoWithSignatures::new(
-            latest_li.ledger_info().clone(),
-            BTreeMap::new(), /* empty signatures */
+        // signatures and invalid data should fail.
+        let bad_li = LedgerInfoWithSignatures::new(
+            LedgerInfo::new(
+                BlockInfo::new(
+                    good_li.epoch(),
+                    0,                 /* round */
+                    HashValue::zero(), /* id */
+                    HashValue::zero(), /* executed_state_id */
+                    good_li.version(),
+                    42, /* bad timestamp_usecs */
```

Comment:
This changes the purpose of the test and renders the name of the test confusing..  I'd rather keep it the old way. Probably add new test just for the "latest li at the same version with my trusted state" case.

Reply:
oh forget to change the name, how about failes_with_invalid_latest_li